aws-rds-dbcluster/resource-role.yaml

```diff
@@ -7,7 +7,7 @@ Resources:
   ExecutionRole:
     Type: AWS::IAM::Role
     Properties:
-      MaxSessionDuration: 8400
+      MaxSessionDuration: 43200
```

Comment:
This is 720 minutes?
Why is this change being made?

Reply:
This is max session duration for IAM role in seconds. This role is being assumed when handlers is executed. The value is set to the [maximum allowed value supported by IAM role](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html).

This file is automatically generated by CloudFomation tooling based on a [resource schema](https://github.com/aws-cloudformation/cloudformation-resource-schema/blob/eeb4064ecff6ebfab325b02096b0bcc83d5d848a/src/main/resources/schema/provider.definition.schema.v1.json#L18-L23). The value for timeout is within the limits provided by CFN platform and the expectation is for CFN platform to manage the sessions to support this value.